internal/provider/provider.go

```diff
@@ -86,6 +86,7 @@ func configure(version string, p *schema.Provider) func(context.Context, *schema
 			TLSClientConfig: &tls.Config{InsecureSkipVerify: d.Get("tls_insecure_skip").(bool)},
 		}}
 		cfg.AddDefaultHeader("Authorization", "apk "+d.Get("key").(string))
+		cfg.AddDefaultHeader("x-dct-client-name", "Terraform")
```

Comment:
Are we not adding Provider version here?

Reply:
so the client name donot contain the version. It just says Terraform. However, for the version we need to refer to the user-agent. More details for this is added [here](https://delphix.atlassian.net/browse/HUBS-1562).